addons/survey/static/src/scss/survey_templates_form.scss

```diff
@@ -473,4 +483,58 @@ _::-webkit-full-page-media, _:future, :root .o_survey_wrap {
     .o_survey_choice_question_skipped {
         color: darken($warning, 10%);
     }
+
+    .o_survey_choice_img img {
+        cursor: default;
+        &:hover {
+            box-shadow: none;
+        }
+    }
 }
+
+/**********************************************************
+    Zoomer Specific Style (surveyChoiceImageZoomer widget)
+    When the width is small (mobile), let space above and below
+    to indicate that the user can close it by clicking out.
+ **********************************************************/
+
+.o_survey_img_zoomer_modal {
+    cursor: pointer;
+    .o_survey_img_zoomer_dialog {
+        background-color: rgba(0,0,0,0.65);
+        @include media-breakpoint-down(sm) {
+            height: 80% !important;
+        }
+        .o_survey_img_zoomer_body {
+            img {
+                max-width: 90%;
+                min-width: clamp(250px, 60%, 450px);
+                max-height: 90%;
+                object-fit: contain;
+            }
+            .o_survey_img_zoomer_close_btn {
+                right: 15px;
+                top: 10px;
+                font-size: 1.4rem;
+                z-index: 1;
```

Comment:
Are those necessary?

Reply:
I moved the font size to the body, no need to have two different indeed. The z-index is, because on diverse situations there can be a bit of overlapping between the image and the close button. (resizing / the image the same ratio as the window can take up to the whole frame)